src/transformers/jest.js

```diff
@@ -26,11 +26,7 @@ export default function transformJestCalls(t, nodePath, mapper, state, cwd) {
   if (moduleArg.node.type === 'StringLiteral') {
     const modulePath = mapper(moduleArg.node.value, state.file.opts.filename, state.opts, cwd);
     if (modulePath) {
-      const newArgs = [...args].map(a => a.node);
-      newArgs[0] = t.stringLiteral(modulePath);
-      nodePath.replaceWith(t.callExpression(
-        calleePath.node, newArgs,
-      ));
+      nodePath.get('arguments.0').replaceWith(t.stringLiteral(modulePath));
```

Comment:
Oh, I didn't know that elements of the `arguments` path were also paths. Will fix that.